manifests/init.pp

```diff
@@ -91,6 +91,12 @@
 #   this parameter with your SSL key's location before deploying this server in a production 
 #   environment.
 #
+# @param default_ssl_reload_on_change
```

Comment:
Thanks for the update of the docs. That actually highlighted something I missed: that it's only the default vhost. Should the name make that explicit?
```suggestion
# @param default_vhost_ssl_reload_on_change
```

Reply:
@ekohl Oh, well, right now i see it: All the default_* parameters in init.pp are also used in vhost.pp for non-default vhosts, as long as there is no own parameter defined for that vhost.
If you agree i would remove the complete "note" i've added to the docs, because it is not correct.